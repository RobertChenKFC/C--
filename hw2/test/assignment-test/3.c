some 
  valid 
    tokens 
   -123 + 45e-67 
    (* .89e101 hello) ! { "This is a string that is not terminated}
