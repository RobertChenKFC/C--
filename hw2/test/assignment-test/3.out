[Error] Lexer: unrecognized token " on line 5
